### ISSUE TITLE
Is there a way to make Serde generate camelCase fields by default?

### ISSUE THREAD

**Issue by cemremengu** [serde-rs/serde#2314]
I am aware that I can use `#[serde(rename_all = "camelCase")]` but I don't want to do this for all structs. Besides there are dynamic/anonymous `Value` objects that I am using.

`camelCase` is one of the most widely used cases and it will help greatly for us (and everyone) when replacing their backends from other technologies (especially from ones like Spring Boot that use camelCase by default)

In short, is there a way to make Serde generate camelCase fields by default?

**Comment by heliping** [comment]
up.

**Comment by heliping** [comment]
+1 up.

**Comment by dtolnay** [comment]
You could convert the data from camelCase to snake_case before deserializing it with serde.

Or you could use a macro that adds a `serde(rename_all = "camelCase")` attribute and then delegates to serde's derives.
